docs/variants/protectli_vp46xx/firmware-update.md

```diff
@@ -9,10 +9,22 @@ firmware version.
 For simplicity we recommend using
 [Dasharo Tools Suite](../../../common-coreboot-docs/dasharo_tools_suite).
 
-## Updating to Dasharo v1.0.18
+## Updating to Dasharo v1.0.18 or v1.0.19
 
-v1.0.18 is a refreshed release, rebased on more up-to-date revision of coreboot
-and it requires to flash whole image as described in [Initial Deployment](initial-deployment.md).
+From v1.0.18 Dasharo firmware is rebased on the more up-to-date revision of
+coreboot.
+
+If the current version of the firmware on the device is older than v1.0.18 or
+you are migrating from proprietary firmware the whole flash chip should be
+flashed as described in [Initial Deployment](initial-deployment.md).
+
+If the current version of the firmware on the device is v1.0.18 and it should
+be updated to v1.0.19, only the `WP_RO` and `RW_SECTION_A` should be flashed.
```

Comment:
Forgot to mention that the Dasahro BIOS Boot medium lock must be disabled beforehand...